File: Store/src/Sales.Domain/Model/Orders/AppliedDiscount.cs
namespace Sales.Domain.Model.Orders
{
    public class AppliedDiscount
    {
        public long DiscountId { get; private set; }
        public long Value { get; private set; }
        internal AppliedDiscount(long discountId, long value)
        {
            DiscountId = discountId;
            Value = value;
        }
    }
}

File: src/main/java/store/sales/domain/model/orders/AppliedDiscount.java
package store.sales.domain.model.orders;

public class AppliedDiscount {
    private final long discountId;
    private final int value;

    public AppliedDiscount(long discountId, int value) {
        this.discountId = discountId;
        this.value = value;
    }

    public long getDiscountId() {
        return discountId;
    }

    public int getValue() {
        return value;
    }
}
